interface/wx/string.h

```diff
@@ -1251,6 +1252,24 @@ class wxString
     */
     bool ToCDouble(double* val) const;
 
+    /**
+        Variant of ToLong()
+
+        Works like ToLong() for signed integers.
+
+        @see ToDouble(), ToLong(), ToULong()
+    */
```

Comment:
```suggestion

@since 3.1.6
    */
```

Reply:
Updated with  fe4b205

Reply:
> Updated with fe4b205

Please feel free to use "Resolve conversation" button in the GitHub web UI instead of commenting.